### PR TITLE
Support for building BiblioSpec libraries from MeroX proxl.xml files.

### PR DESCRIPTION
**Pull request by nickshulman**
Added support for building spectral libraries from MeroX proxl.xml files (requested by Juan)

(still need to add a BiblioSpec unit test)

### CODE REVIEW

**Review by brendanx67 — APPROVED** [review]
Looks okay to me. Obviously, Matt should review, as you have requested.

File: pwiz_tools/BiblioSpec/src/ProxlXmlReader.cpp [hunk 1]
```diff
@@ -306,9 +309,12 @@ double ProxlXmlReader::getScoreThreshold()
 {
     switch (analysisType_)
     {
-        case BYONIC_ANALYSIS: return blibMaker_.getScoreThreshold(BYONIC);
-        case PERCOLATOR_ANALYSIS: return blibMaker_.getScoreThreshold(GENERIC_QVALUE_INPUT);
-        case PLINK_ANALYIS: return blibMaker_.getScoreThreshold(GENERIC_QVALUE_INPUT);
+        case BYONIC_ANALYSIS: 
+			return blibMaker_.getScoreThreshold(BYONIC);
```
Comment (chambm) [inline pwiz_tools/BiblioSpec/src/ProxlXmlReader.cpp:313]: I guess your C# tab settings didn't get carried over to the C++ editor. :)

File: pwiz_tools/BiblioSpec/src/ProxlXmlReader.cpp [hunk 2]
```diff
@@ -107,6 +107,8 @@ void ProxlXmlReader::startElement(const XML_Char* name, const XML_Char** attr) {
                 analysisType_ = BYONIC_ANALYSIS;
             else if (program == "plink")
                 analysisType_ = PLINK_ANALYIS;
+			else if (program == "merox")
```
Comment (chambm) [inline pwiz_tools/BiblioSpec/src/ProxlXmlReader.cpp:110]: I guess your C# tab settings didn't get carried over to the C++ editor. :)

**Review by chambm — APPROVED** [review]
Sorry for the delay. The blib changes look fine except for tabs. I'm looking into the MGF TITLE thing. It looks like SpectrumList::findSpotId does look up spectrum titles. But it requires the full title, not just a substring from it. That can be augmented though to address the MGF scan number issue you emailed about.

**Comment by chambm** [comment]
Hi @nickshulman ,

Looks like you actually added support for this variant of proXL. Do you remember why you picked "qvalue" instead of "score"?

**Comment by brendanx67** [comment]
We usually do use a q value when it is present. We rarely use a native
score. What is "score"? Is it a probability value? If so what kind? e.g
expectation, posterior error probability, etc.

**Comment by chambm** [comment]
MeroX Score. According to Juan it's calculated across multiple files. Looks like it might be a -log score of expectation or probability (higher is better).

**Pull request merged**